### PR TITLE
feat(extension): routes and layout

### PR DESCRIPTION
**Pull request by bakhrom-akbarov**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/extension/CoinView/CoinsList.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+import { CoinDataItem } from './types'
+
+const listItemHeight = 73
+
+const ListRowStyled = styled.div`
+  display: flex;
+  height: ${() => `${listItemHeight}px`};
+`
+
+const Cell: any = styled.div`
```
Comment (TarasPotsikaylo) [inline packages/blockchain-wallet-v4-frontend/src/scenes/extension/CoinView/CoinsList.tsx]: why any?

File: packages/blockchain-wallet-v4-frontend/src/scenes/extension/CoinView/EmptyState.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  height: 100%;
+`
+
+const ButtonStyled = styled(Button)`
+  background: #65a5ff;
+  border-color: #65a5ff;
+  color: ${(props) => props.theme.exchangeLogin};
+`
+
+const EmptyState = () => (
```
Comment (TarasPotsikaylo) [inline packages/blockchain-wallet-v4-frontend/src/scenes/extension/CoinView/EmptyState.tsx:21]: non meanfull name
mb, NoCoins, EmptyCrypto, etc

File: packages/blockchain-wallet-v4-frontend/src/scenes/extension/Intro/index.tsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+          </Text>
+        </div>
+        <Footer>
+          <Button
+            capitalize
+            data-e2e='installAndroidAppBtn'
+            height='48px'
+            fullwidth
+            nature='secondary'
+            // onClick={() => console.log('Click')}
```
Comment (TarasPotsikaylo) [inline packages/blockchain-wallet-v4-frontend/src/scenes/extension/Intro/index.tsx]: remove

File: packages/blockchain-wallet-v4-frontend/src/hooks/useCoinBalances/useCoinBalances.tsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+import { CoinDataItem } from '../../scenes/extension/CoinView/types'
+
+export const useCoinBalances = () => {
+  const state = useSelector((state: RootState) => state)
+  const [coins, setCoins] = useState<CoinDataItem[] | null>(null)
+
+  const allowedChains = ['ETH', 'BTC', 'XLM', 'BCH', 'STX']
+
+  useEffect(() => {
+    setTimeout(() => {
```
Comment (TarasPotsikaylo) [inline packages/blockchain-wallet-v4-frontend/src/hooks/useCoinBalances/useCoinBalances.tsx:16]: add TODO, Need to refactor this code

Comment (bakhrom-akbarov) [reply packages/blockchain-wallet-v4-frontend/src/hooks/useCoinBalances/useCoinBalances.tsx:16]: Why?

File: packages/blockchain-wallet-v4-frontend/src/scenes/extension/CoinView/EmptyState.tsx (hunk 2 above)
Comment (bakhrom-akbarov) [reply packages/blockchain-wallet-v4-frontend/src/scenes/extension/CoinView/EmptyState.tsx:21]: https://www.toptal.com/designers/ux/empty-state-ux-design

**Comment by VitaliiShpital** [comment]
will be recreated with new destination

**Pull request closed without merging**